src/lib/modules/storage/storageProcessesLauncher.js

```diff
@@ -145,6 +153,13 @@ class StorageProcessesLauncher {
         callback();
       });
 
+      self.processes[storageName].on('result', constants.storage.restart, (_msg) => {
```

Comment:
Doesn't look like we're using `_msg` here

Reply:
That's what the `_` does. It tells us and the linter that this variable is not used, but should stay there, so that we know later that we can use it.

Reply:
Fair enough. Didn't know linters would take `_` prefix into consideration. I personally would've just remove the argument that's not used in the first placed.